app/models/program.rb

```diff
@@ -94,4 +95,15 @@ def create_difficulty_levels
                                                 color: '#EF6E69')
     true
   end
+
+  ##
+  # Check if languages string has the right format. Used as before_save.
+  #
+  def check_languages_format
+    # All white spaces are removed to allow languages to be separated by ',' and ', ' but only the first option is saved
+    if self.languages.present?
+      self.languages = self.languages.delete(' ')
+      self.languages.match(/^$|(\A[a-z][a-z](,[a-z][a-z])*\z)/).present?
```

Comment:
In the event only one language is selected, but this code is for the program, where the languages that are allowed for events are defined. Then it is a list of languages. There is a selector when you create a proposal with all the languages in this list